scipy/optimize/_qap.py

```diff
@@ -0,0 +1,418 @@
+                The algorithm may be sensitive to the initial permutation
+                matrix (or search position) chosen due to the possibility
+                of several local minima within the feasible region.
+                With only 1 initialization, a barycenter init will
+                likely return a more accurate permutation.
+                Choosing several random initializations as opposed to
+                the non-informative barycenter will likely result in a
+                more accurate result at the cost of higher runtime.
+                The initial position chosen:
+                "barycenter" : the non-informative "flat doubly stochastic
+                matrix,":math:`J=1*1^T /n` , i.e the barycenter of the
+                feasible region (where n is the number of nodes and '1' is
+                a (n, 1) array of ones)
+                "rand" : some random point near :math:`J, (J+K)/2`, where K
+                is some random doubly stochastic matrix
+            max_iter : int, positive (default = 30)
+                Integer specifying the max number of Franke-Wolfe iterations.
+                FAQ typically converges with modest number of iterations.
+            shuffle_input : bool (default = True)
+                To avoid artificially high or low matching due to inherent
+                sorting of input adjacency matrices, gives users the option
+                to shuffle the nodes of `cost_matrix`. Results are then
+                unshuffled so that returned `col_ind` matches the node order
+                of inputs
+            eps : float (default = 0.05)
+                A positive, threshold stopping criteria such that Franke-
+                Wolfe continues to iterate while Frobenius norm of
+                :math:`(P_{i}-P_{i+1}) > eps`
+
+    Returns
+    -------
+    res : OptimizeResult
+        A :class:`scipy.optimize.OptimizeResult` consisting of the fields:
+
+            col_ind : 1-D array
+                An array of column indices corresponding to the optimal
+                permutation (with the fixed seeds given) of the
+                nodes of `dist_matrix`, to best minimize the objective function
+                :math:`f(P) = trace(A^T PBP^T)`.
+            score : float
+                The optimal value of the objective function.
+            nit : int
+                The total number of Franke-Wolfe iterations performed during
+                optimization.
+
+    References
+    ----------
+    .. [1] J.T. Vogelstein, J.M. Conroy, V. Lyzinski, L.J. Podrazik,
+           S.G. Kratzer, E.T. Harley, D.E. Fishkind, R.J. Vogelstein, and
+           C.E. Priebe, "Fast approximate quadratic programming for graph
+           matching," PLOS one, vol. 10, no. 4, p. e0121002, 2015.
+
+    .. [2] D. Fishkind, S. Adali, H. Patsolic, L. Meng, D. Singh, V. Lyzinski,
+           C. Priebe, "Seeded graph matching", Pattern Recognit. 87 (2019):
+           203–215.
```

Comment:
That "dash" is Unicode 0x2013
```suggestion
           203-215.
```